server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -287,9 +235,15 @@ const orderMutations = {
           throw new Error('Amount is less than minimum value allowed for this Tier.');
         }
 
-        // check if the amount is different from the previous amount
+        // If using a FIXED tier, amount cannot be different from the tier's amount
+        if (tierInfo && tierInfo.amountType === 'FIXED' && amountInCents < tierInfo.amount) {
```

Comment:
Sounds like this should be:

```suggestion
        if (tierInfo && tierInfo.amountType === 'FIXED' && amountInCents !== tierInfo.amount) {
```

However, I'm wondering how this would play with platform tips and taxes (VAT, GST). So maybe your solution is a good way to solve that without too much complication.

Reply:
Good catch, will change!

Reply:
@sbinlondon If you do, make sure you properly test platform tips and tax form. Otherwise we can keep it as it is.